test/stubs/StubRules.ts

```diff
@@ -0,0 +1,93 @@
+import {Rule, RuleType, SeverityLevel} from '@salesforce/code-analyzer-core';
+
+export class StubRule1 implements Rule {
+    public getName(): string {
+		return 'StubRule1';
+    }
+
+    public getEngineName(): string {
+		return 'FakeEngine1';
+    }
+
+    public getSeverityLevel(): SeverityLevel {
+		return SeverityLevel.High;
+    }
+
+	public getFormattedSeverity(): string {
```

Comment:
Seems like this file mixes tabs and spaces. So github review is looking odd.

Reply:
Oh, good catch. I think it's because I copy-pasted some of this code from the `code-analyzer-core` repo, which uses spaces.